extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/MSQTestBase.java

```diff
@@ -1010,6 +1041,23 @@ public void verifyPlanningErrors()
       assertThat(e, expectedValidationErrorMatcher);
     }
 
+    protected void verifyLookupLoadingInfoInTaskContext(Map<String, Object> context)
```

Comment:
Can this be private?

Reply:
No, since it's referred in different subclasses.

Reply:
Sounds good.